lua/graph.lua

```diff
@@ -281,8 +281,8 @@ hazmat = {
 ["designated"] = "true",
 ["yes"] = "true",
 ["no"] = "false",
-["destination"] = "true",
-["delivery"] = "true"
+["destination"] = "false",
+["delivery"] = "false"
```

Comment:
IMHO we shouldn't allow trucks on roads with `hazmat=destination`, as we always allow starting/ending on a `hazmat=no` road, so for us it actually has no meaning.